23c23
<     "hi": "25/7/1980, पू 1:35:33",
---
>     "hi": "25/7/1980, 1:35:33 am",
26c26
<     "ar": "25‏/7‏/1980، 1:35:33 ص",
---
>     "ar": "٢٥‏/٧‏/١٩٨٠، ١:٣٥:٣٣ ص",
30c30
<     "ur": "25/7/1980، 1:35:33 ق.د.",
---
>     "ur": "25/7/1980، 1:35:33 AM",
44c44
<     "ar": "25‏/7‏/1980",
---
>     "ar": "٢٥‏/٧‏/١٩٨٠",
80c80
<     "ar": "275,760.913",
---
>     "ar": "٢٧٥٬٧٦٠٫٩١٣",
116c116
<     "ar": "قبل 586,920.617 ساعة",
---
>     "ar": "قبل ٥٨٦٬٩٢٠٫٦١٧ ساعة",
Index: node-v22.15.1/test/fixtures/icu/localizationData-v74.2.json
===================================================================
--- node-v22.15.1.orig/test/fixtures/icu/localizationData-v74.2.json
+++ node-v22.15.1/test/fixtures/icu/localizationData-v74.2.json
@@ -20,14 +20,14 @@
   "dateTimeFormats": {
     "en": "7/25/1980, 1:35:33 AM",
     "zh": "1980/7/25 01:35:33",
-    "hi": "25/7/1980, पू 1:35:33",
+    "hi": "25/7/1980, 1:35:33 am",
     "es": "25/7/1980, 1:35:33",
     "fr": "25/07/1980 01:35:33",
-    "ar": "25‏/7‏/1980، 1:35:33 ص",
+    "ar": "٢٥‏/٧‏/١٩٨٠، ١:٣٥:٣٣ ص",
     "bn": "২৫/৭/১৯৮০, ১:৩৫:৩৩ AM",
     "ru": "25.07.1980, 01:35:33",
     "pt": "25/07/1980, 01:35:33",
-    "ur": "25/7/1980، 1:35:33 ق.د.",
+    "ur": "25/7/1980، 1:35:33 AM",
     "id": "25/7/1980, 01.35.33",
     "de": "25.7.1980, 01:35:33",
     "ja": "1980/7/25 1:35:33",
@@ -41,7 +41,7 @@
     "hi": "25/7/1980",
     "es": "25/7/1980",
     "fr": "25/07/1980",
-    "ar": "25‏/7‏/1980",
+    "ar": "٢٥‏/٧‏/١٩٨٠",
     "bn": "২৫/৭/১৯৮০",
     "ru": "25.07.1980",
     "pt": "25/07/1980",
@@ -77,7 +77,7 @@
     "hi": "2,75,760.913",
     "es": "275.760,913",
     "fr": "275 760,913",
-    "ar": "275,760.913",
+    "ar": "٢٧٥٬٧٦٠٫٩١٣",
     "bn": "২,৭৫,৭৬০.৯১৩",
     "ru": "275 760,913",
     "pt": "275.760,913",
@@ -113,7 +113,7 @@
     "hi": "5,86,920.617 घंटे पहले",
     "es": "hace 586.920,617 horas",
     "fr": "il y a 586 920,617 heures",
-    "ar": "قبل 586,920.617 ساعة",
+    "ar": "قبل ٥٨٦٬٩٢٠٫٦١٧ ساعة",
     "bn": "৫,৮৬,৯২০.৬১৭ ঘন্টা আগে",
     "ru": "586 920,617 часа назад",
     "pt": "há 586.920,617 horas",
